spikes/aks-secure-baseline/charts/ngsa/values.yaml

```diff
@@ -0,0 +1,66 @@
+# Default values for ngsa.
```

Comment:
This file was copied from ngsa helm chart and modified. References to cert-manager, istio, and keda were removed for the spike.